jbpm-human-task/jbpm-human-task-jpa/src/main/java/org/jbpm/services/task/impl/model/ContentDataImpl.java

```diff
@@ -51,6 +68,53 @@ public void setType(String type) {
 	public void setContent(byte[] content) {
 		this.content = content;
 	}
+	
+	@Override
+	public Object getContentObject() {
+        MarshallerReaderContext context = null;
+        try {
+            ByteArrayInputStream stream = new ByteArrayInputStream(content);
```

Comment:
@mswiderski this method is actually here because implementations like MapDB might just store the whole object entirely, without needing to depend on a marshal/unmarshal mechanism to write the object. In the JPA implementation here, it was previously throwing an UnsupportedOperationException, and you suggested we modified that. Would returning a null value be sufficient here for JPA?